test/e2e/main_test.go

```diff
@@ -30,18 +31,21 @@ type testContext struct {
 	timeout          time.Duration
 	kubeclient       kubernetes.Interface
 	trackerConfigMap *v1.ConfigMap
+	windowsVMs       []types.WindowsVM
```

Comment:
Please add a comment saying there is a dependency on having the connectivity validations tests running for this to be populated.